### ISSUE TITLE
Relax "poison" version dependency (currently unusable with phoenix 1.3.x)

### ISSUE THREAD

**Issue by kwando** [mroth/exmoji#14]
This lib cannot be used with phoenix ```1.3.0```

```
Failed to use "poison" (version 3.1.0) because
  exmoji (version 0.2.2) requires ~> 1.5
  phoenix (version 1.3.0) requires ~> 2.2 or ~> 3.0
  mix.lock specifies 3.1.0
```

**Comment by kwando** [comment]
Using github master works, can we please have a new realease of this :)

**Comment by mroth** [comment]
@kwando it's been a while since I've used Hex (or Elixir even) -- would you mind linking me to a list of steps for what would be required to push master as a new release to Hex? Thank you!

**Comment by mroth** [comment]
Now resolved with v0.3.0
